web/packages/teleterm/src/ui/DeepLinks/deepLinks.test.tsx

```diff
@@ -27,16 +27,21 @@ import { makeRootCluster } from 'teleterm/services/tshd/testHelpers';
 import { App } from 'teleterm/ui/App';
 import { MockAppContext } from 'teleterm/ui/fixtures/mocks';
 
+import { launchDeepLink } from './launchDeepLink';
+
+jest.mock<typeof import('./launchDeepLink')>('./launchDeepLink', () => ({
+  launchDeepLink: jest.fn().mockResolvedValue(undefined),
```

Comment:
I kind of hate how "untestable" React hooks make your code. As long as `launchDeepLink` is defined as a method on a class or a function exported from a hook, I can just completely mock it out like I did here.

However, if `launchDeepLink` was defined in a React hook or a React component, this would be simply impossible.